config/templates/cloud-config-controller

```diff
@@ -380,6 +380,19 @@ write_files:
           "http://127.0.0.1:8080/api/v1/namespaces/kube-system/services"
       done
 
+      {{if .Experimental.Plugins.Rbac.Enabled}}
+      # k8s > 1.5 already have this clusterRole, so we accept http.code != 200
+      mfdir=/srv/kubernetes/rbac
+      post_yaml "@${mfdir}/clusterrole.yaml" \
+      "http://127.0.0.1:8080/apis/rbac.authorization.k8s.io/v1alpha1/clusterroles" |true
```

Comment:
nit but could you print a message when the code isn't 200? probably something like `failed to create the cluster role named "cluster-admin". It's ok if you're using k8s newer than 1.5`